ddev/tests/cli/release/agent/test_integrations_changelog.py

```diff
@@ -0,0 +1,118 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+from datadog_checks.dev.tooling.constants import get_root, set_root
```

Comment:
We shouldn't be using the old code, for tests you can use the `repository` fixture